easybuild/scripts/fix_broken_easyconfigs.py

```diff
@@ -137,9 +138,5 @@ def main():
     for ec_file in ec_files:
         process_easyconfig_file(ec_file)
 
-
-if __name__ == '__main__':
```

Comment:
some stuff was already being executed outside of this anyway, and it's a script, not a module